Source/Core/DiscIO/VolumeWad.cpp

```diff
@@ -60,16 +53,26 @@ IVolume::ECountry CVolumeWAD::GetCountry() const
 	if (!m_pReader)
 		return COUNTRY_UNKNOWN;
 
-	return CountrySwitch(m_Country);
+	// read the last digit of the titleID in the ticket
+	u8 country_code;
+	Read(m_tmd_offset + 0x0193, 1, &country_code);
+
+	if (country_code == 2) // SYSMENU
+	{
+		u16 titlever = 0;
```

Comment:
Could this be called `title_version` (or `title_ver`)? The way it is now, it's a bit hard to see if the word boundary is at "ver" or "ever".